pkg/web_app/lib/src/api_client/_authenticated_client.dart

```diff
@@ -2,23 +2,26 @@
 // for details. All rights reserved. Use of this source code is governed by a
 // BSD-style license that can be found in the LICENSE file.
 
-import 'package:http/browser_client.dart';
-import 'package:http/http.dart';
+import 'dart:async';
+import 'dart:html';
+
+import 'package:http/browser_client.dart' as browser_client;
+import 'package:http/http.dart' as http;
```

Comment:
What are changing in this file?

Reply:
IIRC this is only a move, but it has some leftovers from an experiment where I've used it as part of `api_client.dart`.